cmd/root.go

```diff
@@ -75,7 +75,7 @@ func init() {
 	rootCmd.AddCommand(restoreCmd)
 }
 
-// get one of available config path from environment variable, $HOME/.config, $HOME
+// get one of available config path from environment variable XDG_CONFIG_HOME or default path $HOME/.config
```

Comment:
how about rephrasing this to
`// get the $XDG_CONFIG_HOME environment variable, defaulting to $HOME/.config`

Reply:
That's much better! Looks good to me :)